pyneuroml/utils/__init__.py

```diff
@@ -473,6 +473,61 @@ def rotate_cell(
 
     return newcell
 
+def translate_cell_to_origin(
+    cell: neuroml.Cell,
+    inplace: bool = False,
+) -> neuroml.Cell:
+    """Return a new cell object translated so that the some is at the origin.
+
+    :param cell: cell object to translate
+    :type cell: neuroml.Cell
+    :param inplace: toggle whether the cell object should be modified inplace
+        or a copy created (creates and returns a copy by default)
+    :type inplace: bool
+    :returns: new neuroml.Cell object
+    :rtype: neuroml.Cell
+
+    """
+    soma_seg_id = cell.get_morphology_root()
+    soma_seg = cell.get_segment(soma_seg_id)
+    cell_origin = numpy.array(
+        [soma_seg.proximal.x, soma_seg.proximal.y, soma_seg.proximal.z]
+    )
+
+    if cell_origin[0] == cell_origin[1] == cell_origin[2] == 0:
+        return cell
+    
+    if not inplace:
+        newcell = copy.deepcopy(cell)
+    else:
+        newcell = cell
+
+
+    print(f"Translating {newcell.id} by x:{cell_origin[0]}, y:{cell_origin[1]}, z:{cell_origin[2]}")
```

Comment:
Should this be a debugging comment? `logger.debug`?

Reply:
Perhaps, got it from rotate_cell() function and though it might be useful